src/metaArrow.h

```diff
@@ -97,9 +97,9 @@ class METAIO_EXPORT MetaArrow : public MetaObject
   bool
   M_Read() override;
 
-  float M_Length; // default 1.0
+  float M_Length{}; // default 1.0
```

Comment:
Is the comment (`default 1.0`) wrong?

Reply:
Well, the truth is that the variable was not initialized at all before!  

Fixed now to initialize to
```
float M_Length{1.0}; //default 1.0
```